client/src/i18n/en/report.json

```diff
@@ -479,6 +479,10 @@
       "PAYMENTS" : "Payments",
       "AVERAGE_COST" : "Average Cost",
       "RECOVERY_CAPACITY" : "Recovery Capacity"
+    },
```

Comment:
These keys did not exist in the (fr) file

Reply:
Great catch!  Thanks!